Battle of Mobei and Han-Xiognu war (#121)
TheShadowOfHassen/0-ad-history-encyclopedia-mod: unused/Han_Xiognu_war.md

> The Han dynasty was founded in 206 BC, after centuries of close to continuous fighting. However, only three years prior, the nomadic horse tribes of the Northern Steppes had united in the Xiognu confederacy under the title of Shanyu, the supreme ruler. These tribes had, over the centuries, frequently invaded Han's border regions. At the time, a Xiognu army led by the Shanyu himself had captured several Han bases.'s border regions. At the time, a Xiognu army led by the Shanyu himself had captured several Han bases.

Reviewer: ```suggestion
The Han dynasty was founded in 206 BC, after centuries of close to continuous fighting. However, only three years prior, the nomadic horse tribes of the Northern Steppes had united in the Xiognu confederacy under the title Shanyu, their supreme ruler. These tribes had, over the centuries, frequently invaded Han's border regions. At the time, a Xiognu army led by the Shanyu himself had captured several Han bases.'s border regions. At the time, a Xiognu army led by the Shanyu himself had captured several Han bases.
```

> Wanting to clear away this threat to the Han people, Liu Bang, the founder of the Han dynasty, led an army to attack the Xiognu. The Xiognu at first appeared to retreat, that, however turned out to be a feint. The Xiognu suddenly turned and attacked the Han—at the Battle of Baideng. The Han, having fallen for the trap, suffered a devastating defeat. Han Emperor Liu Bang barely managed to escape to a nearby fortress, which was in turn besieged by the Xiognu. The newly founded empire was on the brink of collapse.

Reviewer: ```suggestion
Wanting to clear away this threat to the Han people, Liu Bang, the founder of the Han dynasty, led an army to attack the Xiognu. The Xiognu at first appeared to retreat, that, however, turned out to be a feint. The Xiognu suddenly turned and attacked the Han at the Battle of Baideng. Suprised, The Han, suffered a devastating defeat. Han Emperor Liu Bang barely managed to escape to a nearby fortress, which was in turn besieged by the Xiognu. The newly founded empire was on the brink of collapse.
```

> But the Han were able to bribe the Shanyu's own wife into convincing the latter to break off the siege and sign a peace treaty instead. This peace treaty obligated the Han to send yearly tribute payments to the Xiognu in order to keep the peace. Although these payments were very high, the Han,  seeing no other choice, paid for the next several decades, even though the Xiognu still frequently plundered Han territory.

Reviewer: ```suggestion
But the Han were able to bribe the Shanyu's own wife into convincing the Xiognu  leader to break off the siege and sign a peace treaty instead. This peace treaty required the Han to send yearly tribute payments to the Xiognu in order to keep the peace. Although these payments were very high, the Han,  seeing no other choice, paid for the next several decades, even though the Xiognu still frequently plundered Han territory.
```

> It was Emperor Wu who finally dared to break the peace. He raised a giant army and planned an ambush at the city of Mayi. A trader was sent to lure the Xiognu into the trap by claiming he had killed the city's ruler and the Shanyu could take it. However, meeting hardly any resistance, the Xiognu got suspicious and eventually caught wind of the plan by questioning a captured Han soldier. The Xiognu turned around and, even though the Han tried to pursue, were able to withdraw without losing a single man. 

Reviewer: ```suggestion
It was Emperor Wu who finally dared to break the peace. He raised a giant army and planned an ambush at the city of Mayi. A trader was sent to lure the Xiognu into the trap by claiming he had killed the city's ruler and the Shanyu could take it. However, meeting no resistance, the Xiognu got suspicious and after questioning a captured Han soldier eventually caught wind of the plan. The Xiognu turned around and, even though the Han tried to pursue, were able to withdraw without losing a single man. 
```

> The two were now officially at war. At first, the Han seemed to stand no chance, as their infantry and chariots proved to be too slow compared to the Xiognu cavalry. And the Chinese had no experience in fighting a nomadic enemy.

Reviewer: ```suggestion
The two were now officially at war. At first, the Han seemed to not stand a chance, as their infantry and chariots proved to be slow compared to the Xiognu cavalry. And the Chinese had no experience in fighting a nomadic enemy.
```

> Over the next few years, the Xiognu would conduct large-scale raids, and the Han were trying to defend themselves as well as possible: They extended and upgraded the Great Wall to counter the Xiognu's hit-and-run tactics. The Han military also shifted focus towards cavalry, raising a cavalry force of hundreds of thousands. For that, general Li Guangli even conducted a whole separate war against a distant kingdom—the War of the Heavenly Horses.

Reviewer: ```suggestion
Over the next few years, the Xiognu would conduct large-scale raids The Han tried to defend themselves as well as possible. They extended and upgraded the Great Wall to counter the Xiognu's hit-and-run tactics. The Han military also shifted focus towards cavalry, raising a cavalry force of hundreds of thousands. For that, general Li Guangli even conducted a whole separate war against a distant kingdom—the War of the Heavenly Horses.
```

> And slowly the tides began turning: Xiognu raids were defended well, and Han commanders like Wei Qing and Huo Qubing were celebrated as heroes (among the Han, of course) for conducting successful counter-raids. They managed to take over the Hexi Corridor, a long, narrow passage leading through an otherwise ungracious and hostile terrain. Controlling it meant being able to trade or tax trade with the Western parts of the world. The Xiognu had lost a crucial support pillar for their economy. Eventually, they retreated northwards behind the Gobi Desert. And the Han, they got back to doing what they did best: building more walls. Extending the existing wall structure along the Hexi Corridor enabled them to open direct trade with the West.

Reviewer: ```suggestion
And slowly the tides began turning: Xiognu raids were fought off, and Han commanders like Wei Qing and Huo Qubing conducted successful counter raids. The Han managed to take over the Hexi Corridor, a long, narrow passage leading through an otherwise ungracious and hostile terrain. Controlling it meant being able to trade with or tax trader from the Western parts of the world. The Xiognu had lost a crucial support pillar for their economy. Bit by bit, they retreated northwards past the Gobi Desert. And the Han built more walls, extending the existing wall structure along the Hexi Corridor, enabled them to open direct trade with the West.
```

> The Han dynasty was founded in 206 BC, after centuries of close to continuous fighting.

Reviewer: ```suggestion
The Han dynasty was founded in 206 BC, after centuries of close to continuous fighting. However, only three years prior, the nomadic horse tribes of the Northern Steppes had united in the Xiognu confederacy under the title of Shanyu, the supreme ruler. These tribes had, over the centuries, frequently invaded Han's border regions. At the time, a Xiognu army led by the Shanyu himself had captured several Han bases.'s border regions. At the time, a Xiognu army led by the Shanyu himself had captured several Han bases.
```

> However, only three years prior, the nomadic horse tribes of the Northern Steppes had united in the Xiognu confederacy under the title of Shanyu, the supreme ruler. These tribes had, over the centuries, frequently invaded border regions. At the time, a Xiognu army led by the Shanyu himself had captured several Han bases.

Reviewer: ```suggestion
```

> Wanting to clear away this threat to the Han people, Liu Bang, the founder of the Han dynasty, led an army to attack the Xiognu. The Xiognu at first gave the impression of retreating, which, however, turned out to be a feint: the Xiognu suddenly turned around and ambushed the Han—at the Battle of Baideng. The Han, having fallen for the trap, suffered a devastating defeat. Han Emperor Liu Bang barely managed to escape to a nearby fortress, which was in turn besieged by the Xiognu. The newly founded empire was on the brink of collapse.

Reviewer: ```suggestion
Wanting to clear away this threat to the Han people, Liu Bang, the founder of the Han dynasty, led an army to attack the Xiognu. The Xiognu at first appearted to retreat, that, however turned out to be a feint. The Xiognu suddenly turned and attacked the Han—at the Battle of Baideng. The Han, having fallen for the trap, suffered a devastating defeat. Han Emperor Liu Bang barely managed to escape to a nearby fortress, which was in turn besieged by the Xiognu. The newly founded empire was on the brink of collapse.
```

> But the Han were able to bribe the Shanyu's own wife into convincing the latter to break off the siege and sign a peace treaty instead.

Reviewer: ```suggestion
```

> This peace treaty obligated the Han to send yearly tribute payments to the Xiognu in order to keep the peace alive. Although these payments were very high, the Han, traumatized by the overwhelming defeat, resorted to doing them for the next several decades. And that in spite of the Xiognu still frequently plundering Han territory.

Reviewer: ```suggestion
But the Han were able to bribe the Shanyu's own wife into convincing the latter to break off the siege and sign a peace treaty instead. This peace treaty obligated the Han to send yearly tribute payments to the Xiognu in order to keep the peace. Although these payments were very high, the Han,  seeing no other choice, paid for the next several decades, even though the Xiognu still frequently plundered Han territory.
```

> It was only Emperor Wu who dared to officially break the peace. He raised a giant army and planned an ambush at the city of Mayi. A trader was sent to lure the Xiognu into the trap by claiming he had killed the city's ruler and the Shanyu could take it. However, meeting hardly any resistance, the Xiognu got suspicious and eventually caught wind of the plan by questioning a captured Han soldier. The Xiognu turned around and, even though the Han tried to pursue, were able to withdraw without losing a single man. 

Reviewer: ```suggestion
It was Emperor Wu who finally dared to break the peace. He raised a giant army and planned an ambush at the city of Mayi. A trader was sent to lure the Xiognu into the trap by claiming he had killed the city's ruler and the Shanyu could take it. However, meeting hardly any resistance, the Xiognu got suspicious and eventually caught wind of the plan by questioning a captured Han soldier. The Xiognu turned around and, even though the Han tried to pursue, were able to withdraw without losing a single man. 
```

The two were now officially at war. At first, the Han seemed to stand no chance, as their infantry and chariots proved to be too slow compared to the Xiognu cavalry. And the Chinese had no experience in fighting a nomadic enemy.
Over the next few years, the Xiognu would conduct large-scale raids, and the Han were trying to defend themselves as well as possible: They extended and upgraded the Great Wall to counter the Xiognu's hit-and-run tactics. The Han military also shifted focus towards cavalry, raising a cavalry force of hundreds of thousands. For that, general Li Guangli even conducted a whole separate war against a distant kingdom—the War of the Heavenly Horses.

> And slowly the tides began turning: Xiognu raids were defended well, and Han commanders like Wei Qing and Huo Qubing were celebrated as heroes (among the Han, of course) for conducting successful counter-raids. They managed to take over the Hexi Corridor, a long, narrow passage leading through an otherwise ungracious and hostile terrain. Controlling it meant being able to trade or tax trade with the Western parts of the world. The Xiognu had lost a crucial support pillar for their economy. Eventually, they retreated northwards behind the Gobi desert. And the Han, they got back to doing what they did best: building more walls. Extending the existing wall structure along the Hexi Corridor enabled them to open direct trade with the West.

Reviewer: ```suggestion
And slowly the tides began turning: Xiognu raids were defended well, and Han commanders like Wei Qing and Huo Qubing were celebrated as heroes (among the Han, of course) for conducting successful counter-raids. They managed to take over the Hexi Corridor, a long, narrow passage leading through an otherwise ungracious and hostile terrain. Controlling it meant being able to trade or tax trade with the Western parts of the world. The Xiognu had lost a crucial support pillar for their economy. Eventually, they retreated northwards behind the Gobi Desert. And the Han, they got back to doing what they did best: building more walls. Extending the existing wall structure along the Hexi Corridor enabled them to open direct trade with the West.
```

> They, however, had not had enough and sought to end this conflict once and for all. Commander Huo Qubing prepared an army and marched as deep into enemy territory as never before, seeking to engage with the Shanyu's army one last time. He marched an arc of over a staggering 1,000 miles, defeating one Xiognu army and capturing many Xiognu nobles on the way. His army reached as far as Lake Baikal. The Shanyu and his army, however, was somewhere else. And not anywhere else. Wei Qing was sent with a smaller army of leftover troops for a smaller expedition further to the west. Against plan, his army encountered the Shanyu's superior elite cavalry, who had been waiting for them, at the Battle of Mobei. Surprisingly, the Han emerged as winners, even annihilating large parts of the Xiognu army. Although the Shanyu escaped this battle, the Xiognu confederacy was ultimately defeated and broke apart.

Reviewer: ```suggestion
The Han, however, were not finished. They wanted to end the conflict once and for all. Commander Huo Qubing prepared an army and marched as deep into enemy territory as never before, seeking to engage with the Shanyu's army one last time. He marched an arc of over a staggering 1,000 miles, defeating one Xiognu army and capturing many Xiognu nobles on the way. His army reached as far as Lake Baikal. But the Shanyu and his army were somewhere else. Wei Qing had been sent with a smaller army of leftover troops for another, smaller, expedition further to the west.  It was his army that encountered the Shanyu's superior force, who had been waiting for them, at the Battle of Mobei. Surprisingly, the Han emerged victorious, managing to destroy large parts of the Xiognu army. Although the Shanyu escaped this battle, the Xiognu confederacy was ultimately defeated and destroyed.
```

> This war, however, had also proven very costly for the Han. Many horses and men had died, the state treasury was drained, and the economy at an long-time low. But all in all, it paid off; the Han profited massively from the new trade routes and were able to maintain peace. And were left alone by the Xiognu (at least for the next century).

Reviewer: ```suggestion
The Han Xiognu war had proven very costly for the Han. Many horses and men had died, the imperial treasury was drained, and the economy at a long-time low. But all in all, it paid off; the Han profited massively from the new trade routes and were able to maintain peace. And were left alone by the Xiognu (at least for the next century).
```

> As for the nomadic tribes, they would go on to migrate westward and remain historically impactful. Those were the ancestors of the Huns, who invaded Europe and, centuries later, caused the downfall of the once-mighty Roman Empire.

Reviewer: ```suggestion
 As for the nomadic tribes, they would go on to migrate westward and remain historically impactful. Those were the ancestors of the Huns, who invaded Europe and, centuries later, caused the downfall of the Roman Empire.
```
